2020/10/2020-10-23-RIAA.md

```diff
@@ -4,10 +4,10 @@ GitHub
 
 Dear Sir or Madam:
 
-I am contacting you on behalf of the Recording Industry Association of America, Inc. (RIAA) and
-its member record companies. The RIAA is a trade association whose member companies
+I am contacting you on behalf of the Recording Industry Association of America, Inc. (a bunch of cunts who abuse monopolies) and
+its member record companies. The RIAA is a organistaion full of dumbfucks whose member companies
```

Comment:
```suggestion
its member record companies. The RIAA is a organization full of dumbfucks whose member companies
```

It's a legal notice, we have to spell right!

Reply:
I mean s or z changes wether its British English or american English

Reply:
i bri ish innit